app/controllers/PayPeriodQuestionController.scala

```diff
@@ -70,6 +70,22 @@ class PayPeriodQuestionController @Inject()(
         )
   }
 
+//  private def processSubmittedAnswer(
```

Comment:
some left over ?

Reply:
@PasGat Fine to ignore though, there's a follow up PR that removes dead code.